src/app.ts

```diff
@@ -45,7 +46,7 @@ export async function getApp(configOverrides: ConfigOverrides = {}) {
   const loggerConfig = resolveLoggerConfiguration(appConfig)
   const enableRequestLogging = ['debug', 'trace'].includes(appConfig.logLevel)
 
-  const app = fastify<http.Server, http.IncomingMessage, http.ServerResponse, pino.Logger>({
+  const app = fastify<http.Server, http.IncomingMessage, http.ServerResponse, FastifyBaseLogger>({
```

Comment:
:green_circle:  Unnecessary import of pino left

Reply:
addressed